string robot_name
string pick_object_name
geometry_msgs/Pose place_pose
---
bool success
---
bool common_area
